src/components/code/utils.tsx

```diff
@@ -27,6 +28,27 @@ const $euiSizeS = 8;
 const isAstElement = (node: RefractorNode): node is AST.Element =>
   node.hasOwnProperty('type') && node.type === 'element';
 
+// Creates an arroy of numbers from comma-separeated
+// string of numbers or number ranges using `-`
+// (e.g., "1, 3-10, 15")
+const parseLineRanges = (ranges: string) => {
+  const highlights: number[] = [];
+  ranges
+    .replace(/ /g, '')
+    .split(',')
+    .forEach((el: string) => {
+      if (el.indexOf('-') > -1) {
```

Comment:
optional, I always think `.includes()` reads more smoothly than `.indexOf` personally
```suggestion
      if (el.includes('-')) {
```